OpenRA.Mods.Common/Traits/Player/HarvesterAttackNotifier.cs

```diff
@@ -57,6 +57,10 @@ void INotifyDamage.Damaged(Actor self, AttackInfo e)
 			if (e.Attacker != null && e.Attacker.Owner == self.Owner)
 				return;
 
+			// 'Damage' is negative (i.e. heal -- e.g. Mutants on Tiberium)
```

Comment:
```suggestion
			// Don't track healing
```

Perhaps change the comment to match the one inthe 56 line